testplan/runnable/interactive/http.py

```diff
@@ -595,27 +595,41 @@ def get(self):
             }
 
         @api.expect(post_export_model)
-        def post(self):
+        def post(self) -> Optional[Dict]:
             save_exports = request.json
             with ihandler.report_mutex:
                 export_context = ExportContext()
                 for exporter in ihandler.exporters:
                     if exporter.name in save_exports.get("exporters", []):
-                        export_result = {
-                            "time": time.time(),
-                            "name": exporter.name,
-                            "uid": strings.uuid4(),
-                        }
                         try:
                             report = copy.deepcopy(ihandler.report)
                             report.reset_uid()
-                            export_path = _run_exporter(
+                            exporter_result = _run_exporter(
                                 exporter=exporter,
                                 source=report,
                                 export_context=export_context,
-                            ).result
-                            export_result["success"] = True
-                            export_result["message"] = export_path
+                            )
+                            export_result = {
```

Comment:
exporter_result v.s export_result (eye crossed, lol)

consider renaming export_result -> request_result? (i'm open to other better names)